src/tags-input.less

```diff
@@ -1,5 +1,6 @@
 .tags-input {
-	display: inline-block;
+	display: inline-flex;
+	flex-wrap: wrap;
```

Comment:
I actually thought we would need `min-width` on the `.tags-input input` - it would make input render on _new line_ if there isn't enough space for it. But I wasn't sure I can come up with a reasonable defaults (percentage, absolute value?). I'll gladly to add whatever defaults you seem appropriate.